modules/datastore/src/Form/DashboardForm.php

```diff
@@ -115,7 +115,7 @@ public static function create(ContainerInterface $container): self {
       $container->get('dkan.metastore.service'),
       $container->get('pager.manager'),
       $container->get('date.formatter'),
-      $container->get('dkan.datastore.service.post_import'),
+      $container->get('dkan.datastore.service.post_import')
```

Comment:
Drupal coding standards say to keep the ending comma: https://www.drupal.org/docs/develop/standards/php/php-coding-standards#:~:text=Note%20that%2C%20as%20seen%20above,end%20of%20the%20list%20later.

Reply:
It's not an array.

Reply:
*heel to forehead. Completely missed that these were function parameters. Of course they are given they are all services. My bad :/